### ISSUE TITLE
Kossel Mini Bed Level Problem

### ISSUE THREAD

**Issue by grimes27** [jcrocholl/kossel#20]
I'm very new to this but, I have my Kossel Mini up and running my only problem is it seems if I run the auto-level before I start printing it doesn't do anything because the extruder is running into the bed.  From all the checking I've done my print bed is very level but if I put my extruder at 1mm above the print bed and move the extruder roughly 40mm in any direction X+, X-, Y+ or Y- it runs into the print bed.  So either I have a convex lens for a printbed, which I doubt, because I have the same problem if I flip the bed around, or the motion of my printer is out of whack.  Any ideas what's going on or how to fix this?

**Comment by kd0aij** [comment]
This is not actually an issue with the design. It sounds more like you skipped a critical part of the calibration process. Look at step #3 here: http://ladvien.github.io/robots/kossel-mini-calibration/ for adjusting delta_radius. (I just did a quick google search; you didn't mention which firmware you are using)

**Comment by grimes27** [comment]
Yes, I didn't have these instructions when I set up my kossel.  I used the
one from Blomker industries and they didn't say anything about the concave
or convex surface.  Thank you very much for this.  I am using Marlin
Firmware too!
